.github/workflows/issue-write.yml

```diff
@@ -39,7 +39,7 @@ jobs:
 
       - name: 'Comment on PR'
         if: steps.download-artifact.outputs.artifact-id != ''
-        uses: actions/github-script@ffc2c79a5b2490bd33e0a41c1de74b877714d736 # v3.2.0
+        uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7.0.1
```

Comment:
This seems fine, but I'm unsure if the omission of breaking changes for v4 means there were none, or that it's just too old to be worth documenting there.

https://github.com/actions/github-script?tab=readme-ov-file#breaking-changes